CHANGELOG.md

```diff
@@ -1,5 +1,26 @@
 # Changelog
 
+## [0.15.1](https://github.com/operator-framework/operator-lifecycle-manager/tree/0.15.1) (2020-05-21)
```

Comment:
Something went wrong here. There are two `# Changelog` headings, and there are two subsections for 0.15.1 with different content.

Reply:
@benluddy updated Changelog.